File: cpp/bc1182.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
	double cVal[12][12], sum = 0.0;
	int col;
	char op;
	
	cin >> col >> op;
	
	for(int i = 0; i < 12; i++)
	{
		for(int j = 0; j < 12; j++)
			cin >> cVal[i][j];
	}
	
	for(int i = 0; i < 12; i++)
		sum += cVal[i][col];
	
	if(op == 'S') cout << fixed << setprecision(1) << sum << endl;
	else cout << fixed << setprecision(1) << sum / 12.0 << endl;
}

File: python/bc1182.py
ma = [[None for j in range(12)] for i in range(12)]
sum = 0.0

col = int(input())
op = input()

for c in range(0, 12):
    for d in range(0, 12):
        ma[c][d] = float(input())

for i in range(0, 12):
    sum += ma[i][col]

if op != "S": sum /= 12

print(f"{sum:.1f}")